### PR TITLE
CakePHP 5 update

### PR DESCRIPTION
**Pull request by LordSimal**
A separate branch for the cake5 changes would be nice 😁

### CODE REVIEW

File: tests/bootstrap.php [hunk 1]
```diff
@@ -23,4 +30,40 @@
 $root = $findRoot(__FILE__);
 unset($findRoot);
 chdir($root);
-require $root . '/vendor/cakephp/cakephp/tests/bootstrap.php';
+
+require_once 'vendor/autoload.php';
+
+define('ROOT', $root . DS . 'tests' . DS . 'test_app' . DS);
```
Comment (ADmad) [inline tests/bootstrap.php]: There's no need to complicate the bootstrapping when just `require $root . '/vendor/cakephp/cakephp/tests/bootstrap.php';` works.

Comment (LordSimal) [reply tests/bootstrap.php]: I was just used to this kind of bootstrapping mechanism to get it working but I have now reverted it.

Comment (ADmad) [reply tests/bootstrap.php]: For most plugins this simple loading is enough, only few need additional/custom config.

**Comment by ADmad** [comment]
Thanks

**Pull request merged**